boards.txt

```diff
@@ -178,13 +178,13 @@ generic.menu.FlashSize.16M15M.build.spiffs_blocksize=8192
 generic.menu.LwIPVariant.open=v2
```

Comment:
It seems that placing lwip2 before v1.4 variants has a side effect of it being used by default.

I don't necessarily mind that (gets tested more, right?), just asking whether that was intended.

Reply:
Yes this is true. In this board.txt and in current master, lwip2 is the default.
And yes lwip2 gets tested more this way. There is an option in the script to change that (```--lwip 1```).
A crash has since been fixed, and the lack of lwip's ping api (declared in espressif SDK) has been noticed but without further issue.

Ldscripts autogeneration is coming in the next few days.